src/Relation/Binary/Construct/Composition.agda

```diff
@@ -0,0 +1,79 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Composition of two binary relations
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Binary.Construct.Composition where
+
+open import Data.Product
+open import Function
```

Comment:
You can get away with importing `Function.Base`.

Reply:
Done